#! /bin/sh
patch -p1 -f -l $* < $0
exit $?

From 0d72b5c7de7f83ac4c44571c73246d2c694c9440 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ingo=20Kl=C3=B6cker?= <dev@ingo-kloecker.de>
Date: Thu, 11 Dec 2025 10:26:18 +0100
Subject: [PATCH] Revert "GIT_SILENT: already done by ecm now"

This reverts commit da45970bd4380598adb3787834867264b894220e.
---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9bec0398..4eb04eb8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -14,7 +14,9 @@ else ()
   set(KF6_WANT_VERSION ${KF_MIN_VERSION})
 endif()

+set(CMAKE_CXX_STANDARD 20)
 set(CMAKE_CXX_SCAN_FOR_MODULES OFF)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)

 find_package(ECM ${KF6_WANT_VERSION} CONFIG REQUIRED)
--
2.52.0
